modules/docs/arrow-docs/docs/docs/effects/resource/README.md

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
this all should be in the kdocs, we no longer create markdown files for data types or type classes